.github/workflows/deploy.yml

```diff
@@ -27,6 +27,11 @@ jobs:
           @fortawesome:registry=https://npm.fontawesome.com/
           //npm.fontawesome.com/:_authToken=${{ secrets.FONTAWESOME_TOKEN_GITHUB }}
           EOF
+      - uses: actions/checkout@v3
```

Comment:
OK, digging a little deeper, I think you can simply add the on line 13 so we're not using the same action twice
```
- name: Use Node.js
  uses: actions/setup-node@v3
  with:
  node-version: '12.x'
```

Reply:
🧠